pandas/tests/io/test_pytables.py

```diff
@@ -4371,11 +4373,11 @@ def test_multiple_open_close(self):
 
             # single
             store = HDFStore(path)
-            assert 'CLOSED' not in str(store)
+            assert 'CLOSED' not in store.info()
```

Comment:
I added a test that str(store) prints the filename, is that what you mean? There's not much behavior in there anymore to test...